core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -350,11 +350,21 @@ private[spark] class AppStatusListener(
         val e = it.next()
         if (job.stageIds.contains(e.getKey()._1)) {
           val stage = e.getValue()
-          stage.status = v1.StageStatus.SKIPPED
-          job.skippedStages += stage.info.stageId
-          job.skippedTasks += stage.info.numTasks
-          it.remove()
-          update(stage, now)
+          // Mark the stage as skipped if in Pending status
```

Comment:
Unnecessary comment. Just repeats the code.

Reply:
Removed.